includes/class-wcpdf-main.php

```diff
@@ -847,35 +851,83 @@ public function set_header_logo_height( $document_type, $document = null ) {
 	}
 
 	/**
-	 * Remove attachments older than 1 week (daily, hooked into wp_scheduled_delete )
+	 * Schedule temporary files cleanup from paths older than 1 week (daily, hooked into wp_scheduled_delete )
 	 */
-	public function attachments_cleanup() {
-		if ( !function_exists("glob") || !function_exists('filemtime') ) {
-			// glob is required
-			return;
+	public function schedule_temporary_files_cleanup() {
+		$cleanup_age_days = isset( WPO_WCPDF()->settings->debug_settings['cleanup_days'] ) ? floatval( WPO_WCPDF()->settings->debug_settings['cleanup_days'] ) : 7.0;
+		$delete_timestamp = time() - ( intval ( DAY_IN_SECONDS * $cleanup_age_days ) );
+		$this->temporary_files_cleanup( $delete_timestamp );
+	}
+	
+	/**
+	 * Temporary files cleanup from paths
+	 * @param  int    $delete_timestamp timestamp
+	 * 
+	 * @return array  Output message
+	 */
+	public function temporary_files_cleanup( $delete_timestamp ) {
+		global $wp_filesystem;
+
+		$delete_before = ! empty( $delete_timestamp ) ? intval( $delete_timestamp ) : null;
```

Comment:
this should default to `time()`, that way it can be used consistently without having to check for `null` values.